expyfun/stimuli/tests/test_tracker.py

```diff
@@ -72,8 +73,8 @@ def test_tracker_ud():
     tr = TrackerUD(None, 3, 1, [1, 0.5], [1, 0.5], 10, np.inf, 1,
                    change_indices=[2])
     tr.respond(True)
-    tr = TrackerUD(None, 3, 1, [1, 0.5], [1, 0.5], np.inf, 10, 1,
-                   change_indices=[2], change_rule='trials')
+    tr = TrackerUD(None, 1, 1, [3, 1.5], [1, 0.5], np.inf, 10, 1,
+                   change_indices=[2], change_rule='trials', x_min=0, x_max=3)
```

Comment:
I suspect these tests do not adequately cover the given functionality. It would be good to construct a track (e.g., 5 or 10 trials?), and check that a warning is emitted, that `check_valid` gives True / False depending on how many reversals are used, and that these behaviors depend on `repeat_limit`.

I know this might seem silly/overkill but such more thorough tests often uncover otherwise hidden bugs.